pkg/operators/klusterlet/controllers/klusterletcontroller/klusterlet_controller.go

```diff
@@ -167,7 +183,25 @@ func (n *klusterletController) sync(ctx context.Context, controllerContext facto
 		return err
 	}
 
-	// Deploy the static resources
+	errs := []error{}
```

Comment:
I wonder how we can reliably test this later with e2e, hrm

Reply:
Maybe we can tag e2es as expected to work under 1.11 and put a CI workflow together that provisions 3.11 - but that might be some wishful thinking

Reply:
i  think it is not easy to e2e this  right now...It will at least need two clusters, hub which is 4.x and spoke  which is 3.11...But i will do test manually.